azure-pipelines.yml

```diff
@@ -78,7 +78,7 @@ jobs:
   - script: |
       set -e
       python -m pip install pandas openml matplotlib
-      python -m pip install "papermill==1.2.1"
+      python -m pip install "papermill"
```

Comment:
Not a big deal, but I think we can remove the quotes if we don't specify the version number (here and below 2 times)